src/Render/md.jl

```diff
@@ -0,0 +1,120 @@
+["Markdown rendering."]
+
+Formats.parsedocs(::Formats.Format{Formats.MarkdownFormatter}, raw, m, obj) = Markdown.parse(raw)
+
+## Prepare methods
+type RenderedMarkdown
+    outdir      :: UTF8String
+    outpages    :: Vector{Tuple{AbstractString, AbstractString}}
+    layout      :: Vector{Tuple{UTF8String, Vector}}
```

Comment:
`outpages` I think could be a dict: but must implement it later and see if it actually works the same.

The other one has to stay ordered.  Anyway I mentioned for you [here](https://github.com/MichaelHatherly/Lexicon.jl/issues/112#issuecomment-108263471) that they plan to implment a OrderedDict / or replace the existing one in 0.4 release.